File: src/test/java/test/ListApplicationSimpleInfoTest.java
package test;
import cn.authing.sdk.java.client.ManagementClient;
import cn.authing.sdk.java.dto.*;
import cn.authing.sdk.java.model.ManagementClientOptions;
import cn.authing.sdk.java.util.JsonUtils;

import java.util.Collections;



public class ListApplicationSimpleInfoTest {

    private static final String ACCESS_KEY_ID = "AUTHING_ACCESS_KEY_ID";
    private static final String ACCESS_KEY_SECRET = "AUTHING_ACCESS_KEY_SECRET";

    public static void main(String[] args) throws Throwable {
        ManagementClientOptions clientOptions = new ManagementClientOptions();
        clientOptions.setAccessKeyId(ACCESS_KEY_ID);
        clientOptions.setAccessKeySecret(ACCESS_KEY_SECRET);
        ManagementClient client = new ManagementClient(clientOptions);

        ListApplicationSimpleInfoDto request = new ListApplicationSimpleInfoDto();
        request.setPage(0);
        request.setLimit(0);
        request.setIsIntegrateApp(Boolean.TRUE);
        request.setIsSelfBuiltApp(Boolean.TRUE);
        request.setSsoEnabled(Boolean.TRUE);
        request.setKeyword("keyword_3512");

        ApplicationSimpleInfoSingleRespDto response = client.listApplicationSimpleInfo(request);
        System.out.println(JsonUtils.serialize(response));
    }

}

File: src/test/go/test/ListApplicationSimpleInfoTest.go
package main

import (
"authing-go-sdk/client"
"authing-go-sdk/dto"
"fmt"
)

func main() {
    options := client.ManagementClientOptions{
        AccessKeyId:     "AUTHING_USERPOOL_ID",
        AccessKeySecret: "AUTHING_USERPOOL_SECRET",
    }
    var err error
    client, err := client.NewClient(&options)
    if err != nil {
        panic(err)
    }
    request := dto.ListApplicationSimpleInfoDto{
        Page: 0,
        Limit: 0,
        IsIntegrateApp: false,
        IsSelfBuiltApp: false,
        SsoEnabled: false,
        Keyword: "keyword_3249",
    }
    response := client.ListApplicationSimpleInfo(&request)
    fmt.Println(response)
}

